service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/ManageHearingsController.java

```diff
@@ -72,8 +76,15 @@ public CallbackResponse handleAboutToStart(@RequestBody CallbackRequest callback
 
         if (isNotEmpty(caseData.getHearingDetails())) {
             caseDetails.getData().put(HEARING_DATE_LIST, hearingsService.asDynamicList(caseData.getFutureHearings()));
+
             caseDetails.getData()
                 .put(PAST_HEARING_LIST, hearingsService.asDynamicList(caseData.getPastAndTodayHearings()));
+
+            if (featureToggleService.isVacateHearingEnabled()) {
```

Comment:
i think toggle is not needed. We can add dynamicList, but it wont be displayed, it is shown only when Vacate hearing is selected on UI, but this option will be hidden

Reply:
ah I had thought this would of thrown a case field not found error since we would not be associated to the event. But It's not toggled off in caseField so it's fine. Good spot.